### ISSUE TITLE
dimension problem

### ISSUE THREAD

**Issue by red-lighting** [laurahanu/2D-and-3D-Deep-Autoencoder#3]
hello
why does it have five deimensions?I know it has x, y, z and the number of patient, but what is the sc?

**Comment by laurahanu** [comment]
The other one is for the number of colour channels e.g. 3 RGB for colour or 1 for b&w

**Comment by red-lighting** [comment]
Excuse me, how do you add the dimension of channel to the 4D matri with x, y, z, the number of patients in Matlab or python. I have created the 4D matri.What's more, do you convert the matri to the file named .npy directly in python?

**Comment by laurahanu** [comment]
There's a numpy function to add an extra dimension: numpy.expand_dims. And yes the .mat file can be coverted to .npy in python

**Comment by red-lighting** [comment]
hello,do you run the code with gpu. I didn't run it in anaconda. I try to configure the theano with gpu, but it mention that there is no pygpu modual. However, there is no pygpu modual that can download in the internet. Did you have trouble in this problem?

**Comment by laurahanu** [comment]
Yes I did use a gpu. Think that's something you have to import/install
